app.js

```diff
@@ -36,6 +36,7 @@ app.use(
 app.use(cookieParser());
 app.use(express.static(path.join(__dirname, "public")));
 app.use(express.static(path.join(__dirname, "utils")));
+// app.use(express.static(path.join(__dirname, "../zegal-client")));
```

Comment:
commented code

Reply:
can we keep that here? I need it while using a local build file

Reply:
removed we will keep these in config